graphblas/operator.py

```diff
@@ -2379,12 +2496,31 @@ def _compile_udt(self, dtype, dtype2):
 
     @classmethod
     def register_anonymous(cls, binaryop, identity, name=None):
+        """
+        Register a Monoid without registering it in the ``graphblas.monoid`` namespace.
+
+        Because it is not registered in the namespace, the name is optional.
+
+        :param BinaryOp binaryop: A builtin or registered binary operator
+        :param identity: The identity value of the monoid
+        :param str name: The name associated with the monoid
```

Comment:
In general, I prefer to mimic what other PyData libraries do such as pandas, dask, xarray, etc.  I rarely see RST in docstrings such as `:param`.  Does the typical way render poorly? e.g.
```
Parameters
----------
binaryop : BinaryOp
```
sphinx+pandoc are pretty flexible with this, aren't they?

Reply:
This style was not working for me, but now that I'm looking at other projects, I think I need to include the `numpydoc` sphinx extension. I agree that the numpy style is easier to read as plain text.

Reply:
Good to know, and thanks!  I really should get more familiar with creating docs...

Reply:
Okay, I got it working for one method. The rendered style is slightly different, but still acceptable. I'll move everything which is autodoc'd over to the numpy parameter style.

Reply:
Awesome, thanks!

We could also consider the google style:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/index.html#google-vs-numpy
but I think likely users/contributed will be more familiar with numpy-style.  I'm open to either.